root/static/scripts/edit/components/Multiselect.js

```diff
@@ -194,7 +194,7 @@ export const MultiselectValue: MultiselectValueComponentT = React.memo(<
         * lost and/or doesn't need to be shifted to an unrelated row;
         * neither situation is accessible.
         */}
-      {state.removed ? l('[removed]') : (
+      {state.removed ? lp('[removed]', 'generic, singular') : (
```

Comment:
Same 1.
For 2. it seems that it can also be an attribute.

```suggestion
      {state.removed ? lp('[removed]', 'generic row') : (
```